src/components/common/hamburger.tsx

```diff
@@ -1,7 +1,19 @@
 import { useState } from "react";
 import SlidingPane from "react-sliding-pane";
 import "./sliding-pane.css";
-import { FaTwitter, FaGithub, FaYoutube, FaLinkedin } from "react-icons/fa";
+import {
+  FaTwitter,
+  FaGithub,
+  FaYoutube,
+  FaLinkedin,
+} from "react-icons/fa";
+import { MdKeyboardArrowDown } from "react-icons/md";
+
+import {
+  Accordion,
+  AccordionHeader,
+  AccordionBody,
+} from "@material-tailwind/react";
 
 const Hamburger = () => {
```

Comment:
Remove old code if not used

Reply:
this is being used. i am using the same hamburg menu you created, just changed the design. already removed unnecessary code, these are required!